lib/datasets.ts

```diff
@@ -110,6 +110,13 @@ export namespace datasets {
         Topk = 'topk',
         Percentiles = 'percentiles',
         Histogram = 'histogram',
+        Variance = 'variance',
+        Stdev = 'stdev',
+        ArgMin = 'argmin',
+        ArgMax = 'argmax',
+        Set = 'set',
```

Comment:
Maybe we should do MakeSet = "makeset" here

Reply:
This must match the server side! So what is it on the server side?

Reply:
After talking to @lilakurse we figured this has to be `makeset`.

Reply:
updated